pyipmi/msgs/sensor.py

```diff
@@ -299,3 +300,27 @@ class GetSensorReadingRsp(Message):
             Optional(UnsignedInt('states1', 1)),
             Optional(UnsignedInt('states2', 1)),
     )
+
+
+@register_message_class
+class PlatformEventReq(Message):
+    __cmdid__ = constants.CMDID_PLATFORM_EVENT
+    __netfn__ = constants.NETFN_SENSOR_EVENT
+    __fields__ = (
+        EventMessageRevision('event_message_rev', 4),
```

Comment:
If comment above is valid we can drop the first parameter.

Reply:
Done